Source/WebCore/style/InlineTextBoxStyle.cpp

```diff
@@ -99,7 +99,7 @@ static const RenderElement* enclosingRendererWithTextDecoration(const RenderText
             return ancestor;
 
         if (!ancestor->isRenderInline()) {
-            // We should always find either the block container or an inline box ancestor inbetween.
+            // We should always find either the block container or an inline box ancestor in between.
```

Comment:
Can we split changes in this file to a separate patch?

Reply:
Oops

Reply:
Fixed